src/components/ButtonDelete/ButtonDelete.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../scss/vars';
+
+.button-delete {
+  a {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: .5em .7em 1em .7em;
+
+    background: $c-warnning;
+    cursor: pointer;
+    clip-path: polygon(0% 0%, 100% 0, 100% 69%, 50% 100%, 0 69%);
```

Comment:
What is this?

Reply:
This is used for "css mask". Basically you can "cut" any element in the shape you want to.  It has a pretty ok browser support these days. It's used here as a progressive improvement and won't affect the UX when the browser doesn't support it.

Reply:
why do you want to cut button?

Reply:
It's just a layout detail which doesn't affect at the accessibility. Trying to make things pretty :D